torch_xla/csrc/ops/gather.cpp

```diff
@@ -0,0 +1,70 @@
+#include "torch_xla/csrc/ops/gather.h"
+
+#include "tensorflow/compiler/xla/xla_client/util.h"
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+xla::XlaOp LowerGather(const xla::XlaOp& input, const xla::XlaOp& index,
+                       xla::int64 dim) {
+  // TODO: Use the new xla::TorchGather() Once it shows up.
```

Comment:
Where would `xla::TorchGather` live? Also: `Once` -> `once`.

Reply:
In tensorflow/compiler/xla/client/lib/slicing.cc

Reply:
Done the Once thing.